source/CMakeLists.txt

```diff
@@ -65,7 +79,10 @@ set_property(TARGET moordyn PROPERTY
 set_property(TARGET moordyn APPEND PROPERTY
   COMPATIBLE_INTERFACE_STRING moordyn_MAJOR_VERSION
 )
-
+if(${CMAKE_VERSION} VERSION_GREATER "3.15") 
+  message("Using Precompiled Headers!")
+  target_precompile_headers(moordyn PRIVATE Misc.hpp)
```

Comment:
I actually never used this... Can we expect issues around inlines and so on? Do not think so, right?

Reply:
My impression from reading about the feature is that it shouldn't behave any differently besides potentially increasing compile times when you change `Misc.hpp` and hopefully decreasing compile times when you change things that include `Misc.hpp`. But I don't have much concrete evidence on its effect on performance especially on compilers other than MSVC, but from what I've experienced it doesn't seem to hurt inlining or performance in other ways.

Reply:
OK, I suppose it is safe to add this. We will always have time to revert it